### ISSUE TITLE
Your project is incredibly awesome!

### ISSUE THREAD

**Issue by xdevelnet** [gh0stwizard/x11mirror-client#1]
Hello!
I could not resist not to create an issue just to write that your x11mirror-* projects are such a good pieces of software!

If you allow me, I have few comments:

1. For my humble opinion, libmicrohttpd is not micro at all. I can't say that it was a surprise for me because of GNU. Even "hello world" they made incredibly complicated.
2. For such a small task as returning image from URL I prefer to NOT use external code/libs. Again, that's just my opinion. You may wish to check out how I solved similar task (http server) at my project.
3. Building x11mirror-server is painful. Really... really painful.
4. > Keylogger for X11

    Wow, this is so sweet :heart_eyes_cat: 
5. It takes some time to understand the whole concept of your project(s). Some people don't want to watch 6 mins video (or videos at all). Maybe you should modify README or I could send you a pull request some day?..

Best wishes, Xdevelnet.

**Comment by gh0stwizard** [comment]
Thanks for the interest of this project!

First of all, I would like to say that the project is a proof of concept. It is not finished and will be never be finished, at least on github, because the form of the finished project is a true malware.

Second. I am just learning the C. I chose libmicrohttpd because it has seemed to been a very easy to use library. And indeed I wrote a prototype of the web-server very quickly.

Third. I knew when I've started the project that there are many ways how to implement the web server part. If you like to use own server -- it's wonderful. That means you have learned basic concepts of the project and you are able to improve it as you would like.

>It takes some time to understand the whole concept of your project(s).

Well, I think I could update README to explain some details.
Thanks again and happy hacking!
